### DISCUSSION TITLE
Static file output

### DISCUSSION THREAD

**Discussion by remram44** [open-telemetry/opentelemetry-collector-contrib#7833] [Q&A]
Hi,

I want to write traces to static files, that I can expose as CI job artifacts. I am already publishing test reports and coverage reports to HTML, and traces don't seem very different. However I don't know if there is a way to do that with opentelemetry?

I noticed that the opentelemetry-collector [can write a JSON file](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/fileexporter). I thought as a stop-gap I could expose that file and load it into Jaeger, but they seem to use different JSON schemas.

Has anyone tried something of the sort?

**Comment by jpkrohling** [comment]
I haven't, but given we have a translator, it shouldn't be hard to convert OTLP's JSON output to Jaeger's.

https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/pkg/translator/jaeger

If you decide to pursue this, let us know once make progress!
